src/plus/launchpad/launchpadProvider.ts

```diff
@@ -1023,17 +1022,27 @@ export function getPullRequestBranchDeepLink(
 	headRefBranchName: string,
 	remoteUrl: string,
 	action?: DeepLinkActionType,
+	prId?: string,
 ) {
 	const schemeOverride = configuration.get('deepLinks.schemeOverride');
 	const scheme = typeof schemeOverride === 'string' ? schemeOverride : env.uriScheme;
+
+	const searchParams = new URLSearchParams({
+		url: ensureRemoteUrl(remoteUrl),
+	});
+	if (action) {
+		searchParams.set('action', action);
+	}
+	if (prId) {
+		searchParams.set('prId', prId);
+	}
+
 	// TODO: Get the proper pull URL from the provider, rather than tacking .git at the end of the
 	// url from the head ref.
 	return Uri.parse(
-		`${scheme}://${container.context.extension.id}/${'link' satisfies UriTypes}/${DeepLinkType.Repository}/-/${
+		`${scheme}://${container.context.extension.id}/${UriTypes.link}/${DeepLinkType.Repository}/-/${
```

Comment:
The question is: should I revert the changes I've made?

IMO, to not increase the bundle size, all enums should be placed in separate filed from logical code. Just enum importing must not be to expensive. But it may be inconvenient from the code structure side

Reply:
reverted

Reply:
reverted